weblogic-azure-aks/src/main/arm/scripts/createVMAndBuildImage.sh

```diff
@@ -158,25 +151,12 @@ export scriptDir="$(cd "$(dirname "${script}")" && pwd)"
 
 source ${scriptDir}/common.sh
 
-export currentResourceGroup=$1
-export wlsImageTag=$2
-export azureACRServer=$3
-export azureACRUserName=$4
-export newImageTag=$5
-export appPackageUrls=$6
-export ocrSSOUser=$7
-export wlsClusterSize=$8
-export enableCustomSSL=$9
-export scriptURL=${10}
-export enableAdminT3Tunneling=${11}
-export enableClusterT3Tunneling=${12}
-export useOracleImage=${13}
-export userProvidedImagePath=${14}
+export newImageTag=$1
```

Comment:
This is a child script, called by different scripts. Keep the positional variable here.